discovery-infra/tests/test_e2e_install.py

```diff
@@ -1,32 +1,40 @@
 import time
-from typing import Tuple
 
 import pytest
+from _pytest.fixtures import FixtureLookupError
 from junit_report import JunitTestSuite
+
 from test_infra.consts import OperatorStatus
 
 from tests.base_test import BaseTest
-from tests.config import ClusterConfig, TerraformConfig
+from tests.config import ClusterConfig
 from tests.conftest import get_available_openshift_versions, get_api_client
 
 
 class TestInstall(BaseTest):
 
+    @pytest.fixture
+    def cluster_configuration(self, request):
```

Comment:
Who calls it?

Reply:
It's a fixture override,  I will document it